policy-build-rust/action.yaml

```diff
@@ -35,3 +35,51 @@ runs:
       shell: bash
       run: |
         kwctl annotate -m ${{ inputs.metadata-file }} -o annotated-policy.wasm ${{ inputs.input-wasm }}
+    -
+      name: Generate the SBOM files
+      shell: bash
+      run: |
+        curl -Lo $RUNNER_TEMP/sbom-tool https://github.com/microsoft/sbom-tool/releases/latest/download/sbom-tool-linux-x64
+        chmod +x $RUNNER_TEMP/sbom-tool
+        $RUNNER_TEMP/sbom-tool generate -D true \
+          -V Verbose \
+          -b ./target/wasm32-wasi/release/ \
+          -bc . \
+          -m . \
+          -nsb https://kubewarden.io \
+          -nsu kubewarden-policy \
+          -pn pod-pribileged-policy \
+          -pv test
+        tmpf=$(mktemp)
+        # remove Microsoft as creator of the file.
+        jq ".creationInfo.creators = [.creationInfo.creators[] | select(. != \"Organization: Microsoft\")]" _manifest/spdx_2.2/manifest.spdx.json > $tmpf
+        tmpf2=$(mktemp)
+        # add Kubewarden team as creator of the file
+        jq ".creationInfo.creators += [\"Organization: Kubewarden\"]" $tmpf > $tmpf2
+        tmpf3=$(mktemp)
+        # fix license set by the tool
```

Comment:
we don't need to do this, this license as I understand it is for the data as output of the sbom-tool. By default is CC0-1.0, which means that we forfeit our copyright on it, and it can be freely used. I think this is ok.
Not needing to track copyright information will help downstream in consumers, if they need to ship this file.

If we change to Apache-2.0, it will retain our copyright, which is just a hassle. Plus, The information in my opinion is not copyrightable anyway, as it is trivially derivative.

Reply:
I've updated the command to update the license of the root package (the policy)